src/pages/NewBeer.jsx

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios'
+import React, { useState } from 'react'
+import Header from '../components/Header'
+
+function NewBeer(props) {
+  const [name, setName] = useState('')
+  const [tagline, setTagline] = useState('')
+  const [description, setDescription] = useState('')
+  const [firstBrewed, setFirstBrewed] = useState('')
+  const [brewersTips, setBrewersTips] = useState('')
+  const [attenuationLevel, setAttenuationLevel] = useState('')
+  const [contributedBy, setContributedBy] = useState('')
+  const [imageUrl, setImageUrl] = useState('')
+  const [success, setSuccess] = useState(false)
+
+  function handleSubmit(event) {
+    event.preventDefault()
+    try {
+      axios.post('https://ih-beers-api2.herokuapp.com/beers/new', {name, tagline, description, first_brewed: firstBrewed, brewers_tips:brewersTips, attenuation_level:attenuationLevel, image_url: imageUrl, contributed_by: contributedBy})
+      .then(result => {
+        setSuccess(true)
+      })
+    }
+    catch (err) {
+      console.log(err)
+    }
+  }
+  return (
+    <div>
+      <Header></Header>
+      <div className='newBeer'>
+        <form onSubmit={handleSubmit} method='POST'>
+          <p>Name</p>
+          <input onChange={e => setName(e.target.value)} name="name"></input>
```

Comment:
Inputs usually are self-closing; that way, you avoid trying to set a value as a child. I would also add the `required` so the user can't send the request without the needed info and break the application. 

Also if you try to send the new beer without the name you can see the app breaking and one of the reasons for that is the `.catch()` can be helpful

```suggestion
          <input onChange={e => setName(e.target.value)} name="name"  type="text" required/>
```